Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,67 @@
+OEM Component Subsystem- Auto Light
+------
+
+## The function of the Auto Light Subsystem
+The Master microcontroller will receive the output of the Nissan Leaf AutoLight in this part of the subsystem. Furthermore, the Master microcontroller will detect if the headlight is off, on, blinking, and high and low beams.
+
+### Specification & constraints
+   - Must detect high beam,low beam, left indicator, and right indicator
+   - Must use OEM componets as the inputs
+   - The signal input must not exceed 5 V 
+   - Implement a way to protect the signal from voltage varying from 10 V to 15 V
+   - Current to the signal must be less than 1 percent of the current to the light
+   - Prevent interference among signals through isolation methods
+   
+# Auto Light Wiring schematic
+![opel2437](https://user-images.githubusercontent.com/101354063/205468261-7fca3396-009b-4eb8-9960-962065d22ac8.png)
+### Figure 1: Headlamp(High and low Relay)
+<br/>
+
+## Design: A Voltage divider from Auto Light to the Microcontroller
+![Capture](https://user-images.githubusercontent.com/101354063/205472902-57cf6121-181d-4fe0-904c-63295cc9c374.PNG)
+
+### Figure 2: Voltage divider
+
+
+<br/>
+
+![image](https://user-images.githubusercontent.com/101354063/206358241-fb6e9602-a89a-4aee-93f3-84056c43c628.png)
```

Comment:
Consider what will happen if a short occurs at the wires connected to the coupler. the conductor overcurrent protection from the car is a higher amperage than typical control signal wires. So, either you will need to use wires that are sufficient to handle the current which is the max correct before the fuse blows or you will need to include a fuse to protect your wiring and ensure that a fire won't occur.

Reply:
I add a 5 mA fuse into the design to prevent overcurrent.